techniques/system/distributePolicy/1.0/rudder-rsyslog-root.st

```diff
@@ -70,3 +70,17 @@ $template RudderReportsFormat,"insert into RudderSysEvents (executionDate, nodeI
 
 ## Syslog messages from "rudder" forwarded from AIX
 :msg, ereregex, "from .*: rudder:" ~
+
+&if(SKIPIDENTIFY)&
+
+$AllowedSender UDP, &ALLOWEDNETWORK&
+$AllowedSender TCP, &ALLOWEDNETWORK&
```

Comment:
&ALLOWEDNETWORK& is empty and in fact never used.

The correct variable to use is &AUTHORIZED_NETWORKS& and is a proper array